k8s/spinnaker/templates/spinnaker_application_manifest_middle_secured.yaml

```diff
@@ -1,6 +1,6 @@
     notes: |-
       # Manage your Spinnaker
-      [Open Cloud Shell](https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy.git&cloudshell_working_dir=k8s/spinnaker/scripts/manage&cloudshell_tutorial=landing_page_expanded.md)
+      [Open Management Environment in Cloud Shell](https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy.git&cloudshell_working_dir=k8s/spinnaker/scripts/manage&cloudshell_tutorial=landing_page_expanded.md)
```

Comment:
@dorbin Might be a bit long :-( Any suggestions?

Reply:
Hm, not terrible?

![image](https://user-images.githubusercontent.com/223702/57948604-1990ad80-78b0-11e9-8bbc-9cd023451715.png)

Reply:
The current one I'm seeing live just says "Open Cloud Shell." 
How 'bout "Open Cloud Shell to manage Spinnaker"?

Reply:
That would also work, but I'd like to try to make it clear that Cloud Shell is an implementation detail, and what we're actually pointing users to is some kind of management interface. I.e. the link should be able to remain the same if we e.g. reimplemented this _without_ Cloud Shell in future.